hail/src/main/scala/is/hail/expr/ir/lowering/LowerTableIR.scala

```diff
@@ -803,52 +807,69 @@ object LowerTableIR {
       case TableKeyByAndAggregate(child, expr, newKey, nPartitions, bufferSize) =>
```

Comment:
Could you fix or delete the todo on the above line? (Not sure if it still ignores nPartitions)